js/modules/k6/experimental/tracing/options.go

```diff
@@ -9,13 +9,14 @@ import (
 // tracing.instrumentHTTP() method.
 type options struct {
 	// Propagation is the propagation format to use for the tracer.
-	Propagator string `js:"propagator"`
+	Propagator string `json:"propagator"`
 
-	// Sampling is the sampling rate to use for the tracer.
-	Sampling *float64 `js:"sampling"`
+	// Sampling is the sampling rate to use for the
+	// tracer, expressed in percents: 0 <= n <= 100.
+	Sampling *float64 `json:"sampling"`
```

Comment:
Is this a pointer so that you can just take this from goja and know if it was set?

If so I kind of prefer if this is parsed from the goja value into a none pointer and then not actually need to access it as a pointer. But this can be done later as well

Reply:
It's a pointer to differentiate between unset field and zero value field. That's the only sane way of doing it in Go that I know 🤓 

Do I understand your suggestion correctly: switch back to a value field instead of a pointer field, have an options constructor that takes a Goja value (holding options) as a default, and manage whether that specific field is bullish explicitly there?

Reply:
This also can enable warnign users on providing unsupported options so that if they misspell something it is easier to find out.

But this can be done in a separte PR, no need to do this now.

Reply:
I addressed it by having a `newOptions` constructor which builds options from a `goja.Value` and defaults to value `1.0` (100%) if the field is absent/nullish. The field is not a pointer anymore 👍🏻